### ISSUE TITLE
Do not convert/calculate on custom variable content

### ISSUE THREAD

**Issue by log1-c** [Icinga/icingadb-web#860]
## Describe the bug
We user the custom variable/field `notification_delay` on service checks to delay notifications for specific kinds of checks.
When entering big numbers like `2592000` they get converted when shown in the webinterface
![image](https://github.com/Icinga/icingadb-web/assets/24474580/5e905eef-5a61-4e40-8245-207bc54bac49)

## Expected behavior
Show the content of the variable as is

## Your Environment
Include as many relevant details about the environment you experienced the problem in.

* Icinga DB Web version (System - About): 1.0.2
* Icinga Web 2 version (System - About): 2.11.4
* Web browser: Edge/Vivaldi
* Icinga 2 version (`icinga2 --version`): 2.14.0
* Icinga DB version (`icingadb --version`): 1.1.0
* PHP version used (`php --version`): 
* Server operating system and version: rhel8

**Comment by nilmerg** [comment]
Seems to be related to the PHP version. I cannot reproduce this on 8.x, only on 7.x.

**Comment by Al2Klimov** [comment]
Given you're right, whatever PHP is doing here is scary:

```
MariaDB [icingadb]> select flatvalue from customvar_flat where flatname='dummy_statee'\G
*************************** 1. row ***************************
flatvalue: 2e+18
1 row in set (0.001 sec)

MariaDB [icingadb]>
```

**Comment by Al2Klimov** [comment]
Yep, in Go `fmt.Sprintf("%v", 2000000000000000000) == "2e+18"`. Our Flatten() uses this.
